driver/normalizer/normalizer.go

```diff
@@ -50,8 +53,11 @@ var Preprocessors = []Mapping{
 	Map(
 		Part("_", Obj{
 			uast.KeyType: String("RegExpLiteral"),
-			"extra": Obj{
-				"raw": Var("raw"),
+			"extra": Fields{
+				{Name: "raw", Op: Var("raw")},
+				//FIXME(bzz): map parenthesis
```

Comment:
I don't think we need to map tokens like parenthesis.

Reply:
Agree with @juanjux, this is merely a TODO. We can add a custom position field into the node and that's it. The priority of this "fix" is very low.

Reply:
This is good suggestion! It's just that in order for it to be consistent between all drivers, I belive we still need to act up on this change and _at least_ document this approach, so it can be referred/followed in the other drivers.

This is motivation to keep the FIXME and make another pass on all drivers addressing those.

Reply:
changed to `TODO`